core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -167,3 +173,57 @@ fun LpOutlinedTextFieldNumber(
         keyboardOptions = KeyboardOptions.Default.copy(keyboardType = KeyboardType.Number)
     )
 }
+
+/**
+ * This function creates a phone number OutlinedTextField
+ * @param modifier Set component modifier
+ * @param value Current phone value
+ * @param showError This parameter determines whether the error is displayed or not
+ * @param onPhoneChange Returns value typed
```

Comment:
Please add the missing params like `imaAction`, `keyboardType` and `isEnabled` with the respective descriptions